### DISCUSSION TITLE
Passing header to rtsp request

### DISCUSSION THREAD

**Discussion by MDalprato** [mpromonet/webrtc-streamer#596] [General]
Hi All,

Is it possibile to pass an additional header to the rtsp request made by the webrtc streamer ?
I need to add the "Range: clock-" header request in order to set the start time of the rtsp stream via ONVIF-G.
thanks

**Comment by mpromonet** [comment]
Hi @MDalprato 

You should be able to set time range setting argument "starttime" or "ntptime" in "options" parameters.

Best Regards,
Michel.
